function_examples/use-env/README.md

```diff
@@ -1,5 +1,5 @@
 # Netlify Function Examples
-Basic Examples of using environment variables inside of Netlify functions
 
-_at this moment you can only use variables specified in the UI and not in netlify.toml_
+This is a basic example of a function that uses a build environment variable that's specified in the users Netlify site's deploy settings.
 
+_at this moment you can only use variables specified in the UI and not in netlify.toml_
```

Comment:
Capitalize the `a` at the beginning and add a period at the end.

Also, this may no longer be true. @calavera could probably confirm.